tests/Client/UpdaterTest.php

```diff
@@ -4,9 +4,54 @@
 
 use PHPUnit\Framework\TestCase;
 use Tuf\Client\Updater;
+use Tuf\Tests\DurableStorage\InMemoryBackend;
 
 class UpdaterTest extends TestCase
 {
+    /**
+     * @return Updater
+     */
+    protected function getSystemInTest()
+    {
+        $mirrors = [
+            'mirror1' => [
+                'url_prefix' => 'http://localhost:8001',
+                'metadata_path' => 'metadata',
+                'targets_path' => 'targets',
+                'confined_target_dirs' => [],
+            ],
+        ];
+
+        // Memory storage used so tests can write without permanent side-effects.
+        $localRepo = $this->populateMemoryStorageFromFixtures('tufclient/tufrepo/metadata/current');
+        $updater = new Updater('repo1', $mirrors, $localRepo);
+        return $updater;
+    }
+
+    public function populateMemoryStorageFromFixtures($path)
+    {
+        $realpath = realpath(__DIR__ . "/../../fixtures/$path");
+        if ($realpath === false || !is_dir($realpath)) {
+            throw new \RuntimeException("Client repository fixtures directory not found at $path");
+        }
+
+        $storage = new InMemoryBackend();
+
+        $iter = new \FilesystemIterator(
+            $realpath,
+            \FilesystemIterator::SKIP_DOTS | \FilesystemIterator::KEY_AS_FILENAME
+        );
+        foreach ($iter as $filename => $info) {
```

Comment:
`$iter` is not a great variable name. Usually I prefer to use whole words rather than abbreviations. Maybe `$fsIterator` or something?